scoring/templates/scoring/includes/advanced-filter.html

```diff
@@ -6,6 +6,7 @@ <h4 class="filter-title">Select which criteria you would like to filter:</h4>
         <button class="close-icon popup-modal__close" type="button"></button>
     </div>
     <form>
+        {% if authority_type != 'northern-ireland' and authority_type != 'combined' %}
```

Comment:
I _think_ (check with @ajparsons!) that you can add `district` and `county` to this list – ie: we only display the England/Scotland/Wales choice for Single Tier councils, because all of the districts and counties are in England.

Reply:
I don't quite understand the issue, but yes, there are no district/county councils outside of England.